src/Trakx.Copper.ApiClient.Tests/Unit/ApiKeyCredentialsProviderTests.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Security.Cryptography;
+using System.Text;
+using System.Threading.Tasks;
+using FluentAssertions;
+using Microsoft.Extensions.Options;
+using NSubstitute;
+using Trakx.Copper.ApiClient.Utils;
+using Trakx.Utils.DateTimeHelpers;
+using Trakx.Utils.Extensions;
+using Xunit;
+
+namespace Trakx.Copper.ApiClient.Tests.Unit
+{
+    public class ApiKeyCredentialsProviderTests
+    {
+        private readonly ApiKeyCredentialsProvider _apiKeyCredentials;
+        private readonly CopperApiConfiguration _configuration;
+        private readonly DateTimeOffset _timestamp;
+        public ApiKeyCredentialsProviderTests()
+        {
+            _configuration = new CopperApiConfiguration { ApiKey = "apikeyPublic", ApiSecret = "c2FsdXRhdG91cw==", BaseUrl = "http://baseUrl.com" };
```

Comment:
My heart rate jumps everytime I see some realistic private key here 😹 I need to chill out !!!
![Chill](https://media.giphy.com/media/fVtFdbjQzWUAczqG9g/giphy.gif)